pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
+        path = lambda x, cascade=path: cascade(
+            '/'.join(x.split(os.path.sep))
+        )
+
+    return path
+
+def _path_resolver(relative_path=True):
+    '''
+    Returns a function that manipulates a path string to be relative if
+    `relative_path' is true, otherwise it returns the identity function.
```

Comment:
Ok for moving in a separate function (if still needed).
Ok to rename this argument - indeed the name I chose is not clear. Sadly, it is already API (callgrind and annotate methods are public), so it can't change there (at least not as a simple renaming).